tenxutils/R/Plot.R

```diff
@@ -578,7 +578,7 @@ markerComplexHeatmap <- function(seurat_object,
   # set up the cluster color palette
   nclust <- length(unique(clusters))
   cluster_cols <- gg_color_hue(nclust)
-  names(cluster_cols) <- 0:(nclust-1)
+  names(cluster_cols) <- sort(as.numeric(as.character(unique(clusters))))
```

Comment:
Hi @mjgomezv,
Thanks for the update! But I'm afraid maybe a problem is introduced here? Because since I pulled it I get an error with colour mapping when drawing the heatmap in summariseMarkers.R saying
"Making a heatmap of the top marker genes from each cluster                                                                                                         Loading required namespace: Cairo                                                                                                                                  Error: cluster: cannot map colors to some of the levels: 4"
Where 4 is always the number of my last cluster, whatever number that may have. 
I've tried but I can't seem to figure out what causes the problem, would you have an idea?

Thanks!
Erinke